wp-content/themes/amplify/amplify-feed.php

```diff
@@ -4,6 +4,33 @@
  * Template Name: Amplify Feed Template
  * Description: Shows the saved links feed that can be embedded on external sites.
  */
+
+if( $_GET['horizontal'] && 'true' == $_GET['horizontal'] ){
+    $body_class = 'horizontal';
+} else {
+    $body_class = 'vertical';
+}
+
+if( $_GET['show_link_descriptions'] && 'true' == $_GET['show_link_descriptions'] ){
+    $saved_links_class = 'show-link-description';
+} else {
+    $saved_links_class = '';
+}
+
+if( $_GET['hide_header'] && 'true' == $_GET['hide_header'] ) {
+    $hide_header = 'hide-header';
+} else {
+    $hide_header = '';
+}
+
+if( $_GET['rows'] && 2 == $_GET['rows'] ){
```

Comment:
these `if ( $variable && 'value' === $variable ) {` checks should be `if ( isset( $variable ) && 'value' === $variable ) {`, to avoid the following warnings:

> Notice: Undefined index: show_link_descriptions in /wp-content/themes/amplify/amplify-feed.php on line 14
> 
> Notice: Undefined index: hide_header in /wp-content/themes/amplify/amplify-feed.php on line 20

Reply:
@benlk These are updated in https://github.com/INN/umbrella-inndev/pull/131/commits/859e6e49e047812a7ca3305d8ddb0800cf9e0172